packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -35,7 +35,7 @@ export async function ensureExtensionsIds(
 
   for (const pending of matchExtensions.toConfirm) {
     // eslint-disable-next-line no-await-in-loop
-    const confirmed = await matchConfirmationPrompt(pending.local, pending.remote)
+    const confirmed = options.force || (await matchConfirmationPrompt(pending.local, pending.remote))
```

Comment:
I was surprised to see that this wasn't here already. Any reason why not?

Reply:
it could potentially break stuff. I think @Arkham was intentional here, if we are not sure about a match, is dangerous to force it

Reply:
Wouldn't people expect `-f` to skip all prompts and accept the consequences? I've had to add this because in `dev` I was getting the deploy match confirmation prompt. I suppose I could solve this by changing the prompt messaged based on wether it's being used by `dev` or `deploy`.

Reply:
now that we are working on running the CLI as a service in spin we should consider an alternative to avoid prompting the user for any confirmation otherwise it won't work as expected, what do you think? @isaacroldan @matteodepalo @Arkham

Reply:
Could we just throw instead of prompting if `force` is true?